services/web/server/src/simcore_service_webserver/catalog/_handlers.py

```diff
@@ -142,7 +139,6 @@ async def dev_get_service(request: Request):
     f"{VTAG_DEV}/catalog/services/{{service_key}}/{{service_version}}",
     name="dev_update_service",
 )
```

Comment:
Q: when do you plan to remove the /dev api entrypoint?

Reply:
As soon as the front-end is in, we will make a PR including the change in the front-end and backend